games-engines/odamex/odamex-0.8.1.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+WX_GTK_VER="3.0"
+inherit cmake-utils desktop wxwidgets xdg
+
+DESCRIPTION="Online multiplayer, free software engine for Doom and Doom II"
+HOMEPAGE="https://odamex.net/"
+SRC_URI="mirror://sourceforge/${PN}/Odamex/${PV}/${PN}-src-${PV}.tar.bz2"
+
+LICENSE="GPL-2+ MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+client master upnp +odalaunch portmidi server"
+REQUIRED_USE="
+	|| ( client master server )
+	portmidi? ( client )
+	upnp? ( server )"
+
+RDEPEND="
+	client? (
+		media-libs/libpng:0=
+		media-libs/libsdl2[X]
+		media-libs/sdl2-mixer
+		odalaunch? ( x11-libs/wxGTK:${WX_GTK_VER}[X] )
```

Comment:
I was going to say `X` is wrong here too but you can keep it. The wxGTK really needs a `gui` flag so that you can ensure that at least `X` or `aqua` are enabled.

Reply:
Should I just leave this as `X` then for now?

Reply:
Actually, I could submit a patch for **x11-libs/wxGTK** adding a `gui` flag that simply requires either `X` or `aqua` to be set.

Reply:
You did it exactly how I would have done it but we'll need to check with @leio first.

Reply:
```
06:12  juippis: leio: https://github.com/gentoo/gentoo/pull/12863/commits the first commit is for you to ack 
09:29  leio: juippis: sounds like a NAK, but I can't deal with it
09:31  leio: ..right now
09:31  leio: if that consumer really works with aqua too, it can do it with its own use flag then and [aqua?,X?] or something
09:32  leio: wxGTK doesn't provide a gui, that's conceptually wrong to start with
09:32  leio: yes, it provides (or doesn't provide) a gui library toolkit, but not a gui :P
09:32  leio: USE=gui is for (optional) GUI, e.g. on top of cli tools
```

(I dont have any word about this PR, thanks @chewi for taking care of it, BUT:) I kinda agree forcing everyone to rebuild wxGTK due to some requirement in *odamex* feels a bit overkill. I believe you should drop the wxGTK update from this PR, and open a bug to discuss about implementing it in future, to not let it block the update for now.

Reply:
I've removed the wxGTK patch and reverted the Odamex back to requesting **x11-libs/wxGTK** with the `X` flag. I've also opened a bug to discuss implementing a `guitk` USE flag or similar: https://bugs.gentoo.org/693636